code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -538,7 +538,7 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 		log_game("DYNAMIC: Starting a round of forced extended.")
 		return TRUE
 	var/list/drafted_rules = list()
-	for (var/datum/dynamic_ruleset/roundstart/rule in roundstart_rules)
+	for (var/datum/dynamic_ruleset/roundstart/rule in shuffle(roundstart_rules))
```

Comment:
This simply adds all of the rules into a list of drafted rules which uses a pickweight function to select the actual applied rule. This doesn't trigger the chosen ruleset, it simply filters them thus shuffling this list has no impact on the rules chosen. See line 556 just below for more info.

Reply:
It appears the only line 541 doesn't need to be shuffled due to line 556, but others don't seem so.
Are you fine with other lines being suffled?